x-pack/plugins/enterprise_search/public/applications/__mocks__/kea.mock.ts

```diff
@@ -110,8 +111,10 @@ export class LogicMounter {
   };
 
   // Automatically reset context & mount the logic file
-  public mount = (values?: object) => {
+  public mount = (values?: object, props?: object) => {
```

Comment:
Just want to quickly note - I'm not a _super_ huge fan of how logic files that don't want defaults and only want props (e.g., AppLogic) now have to either pass `undefined` or `{}` as a first param. If I could go back in time I'd probably change this to

```js
  public mount = ({ defaults, props }) => {
```

and have test files pass in their API like `mount({ defaults: { foo: 'bar' } })` or `mount({ props: { bar: 'baz' } })`. IDK if we feel strongly to make that change in a ton of files at this point so I'm fine leaving it as-is, mostly just thinking out loud

Reply:
Also for more context, here's the first conversation where I mentioned LogicMounter need to support props: https://github.com/elastic/kibana/pull/92560#discussion_r583018105

Reply:
I supposed you _could_ do a `mountWithProps`, which would not require the empty object, but I am not a huge fan of that.

Reply:
Ya for sure, to clarify I don't hate this either and I'm fine with shipping it for now - we can definitely come back to his later if we start using a ton more logic files w/ props!